app/views/api/v2/hosts/show.json.rabl

```diff
@@ -25,3 +25,16 @@ end
 child :config_groups do
   extends "api/v2/config_groups/main"
 end
+
+host_additional_views(@host).each do |id, tab|
+  if tab.is_a? String
+    node do |host|
+      partial "api/v2/#{tab}", :object => host
```

Comment:
This doesn't actually work with namespaced views, which you generally do in an engine.  we'd need 'katello/api/v2/#{tab}'

I dont' know that i'm a huge fan of re-using the UI 'tab' mechanism for api requests as well.  I don't think a 'tab' in the ui would necessarily correspond to a rabl in the json.  Also for the katello case at first we don't even want to add any UI tabs but do need the parts exposed in the API.